modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/resources/META-INF/resources/js/edit_blueprint/Sidebar.js

```diff
@@ -30,11 +30,22 @@ const EmptyListMessage = () => (
 	</div>
 );
 
-const QueryFragmentList = ({category, onAddFragment, queryFragments}) => {
+const DEFAULT_EXPANDED_LIST = ['match'];
+
+const QueryFragmentList = ({
+	category,
+	expand,
+	onAddFragment,
+	queryFragments,
+}) => {
 	const {locale} = useContext(ThemeContext);
 
 	const [showAdd, setShowAdd] = useState(-1);
-	const [showList, setShowList] = useState(true);
+	const [showList, setShowList] = useState(expand);
+
+	useEffect(() => {
+		setShowList(expand);
+	}, [expand]); //eslint-disable-line
```

Comment:
Why does eslint need to be disabled here?

Reply:
Oops, I'll take it out!